backend/src/database/migrations/20220915123536-create-email-messages-transactional.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const { TransactionalEmailMessageStatus } = require('@email/models')
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('email_messages_transactional', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT,
+      },
+      user_email: {
+        type: Sequelize.DataTypes.TEXT,
+        allowNull: false,
+        references: {
+          model: 'users',
+          key: 'email',
+        },
+        onUpdate: 'CASCADE',
+      },
+      from_name: {
+        allowNull: false,
+        type: Sequelize.DataTypes.STRING(255),
+      },
+      from_address: {
+        allowNull: false,
+        type: Sequelize.DataTypes.STRING(255),
+      },
+      recipient: {
+        allowNull: false,
+        type: Sequelize.DataTypes.STRING(255),
+      },
+      params: {
+        type: Sequelize.DataTypes.JSON,
+        allowNull: false,
+      },
+      message_id: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      attachments_metadata: {
+        type: Sequelize.DataTypes.ARRAY(Sequelize.DataTypes.JSON),
+        allowNull: true,
+      },
+      status: {
+        type: Sequelize.DataTypes.ENUM(
+          TransactionalEmailMessageStatus.Unsent,
+          TransactionalEmailMessageStatus.Accepted,
+          TransactionalEmailMessageStatus.Sent
```

Comment:
Can we have these bunch as well so we don't have to make another migration when we start handling the other statuses? As the
![Screenshot 2022-10-02 at 12 26 50 PM](https://user-images.githubusercontent.com/12974927/193437861-c54f2aa9-ff6c-4a54-a50e-58de998e0db8.png)
y're already confirmed from our RFC